src/components/deploymentConfig/DeploymentTemplateView.tsx

```diff
@@ -147,7 +145,7 @@ export const ChartTypeVersionOptions = ({
     return (
         <div
             className={`chart-type-version-options pr-16 pt-8 pb-8 ${
-                disableVersionSelect || selectedChart?.name !== ROLLOUT_DEPLOYMENT ? '' : 'dc__border-right'
+                disableVersionSelect || selectedChart?.name !== ROLLOUT_DEPLOYMENT && selectedChart?.name !== DEPLOYMENT ? '' : 'dc__border-right'
```

Comment:
we should write `disableVersionSelect || (selectedChart?.name !== ROLLOUT_DEPLOYMENT && selectedChart?.name !== DEPLOYMENT)`